src/passes/Asyncify.cpp

```diff
@@ -1555,6 +1561,28 @@ struct Asyncify : public Pass {
                                          Builder::Mutable));
   }
 
+  void addImportedGlobals(Module* module) {
+    Builder builder(*module);
+    {
+      auto asyncify_state = builder.makeGlobal(ASYNCIFY_STATE,
```

Comment:
`asyncifyState` (camelCase) is more consistent I think, for the source code (even though the wasm name might be different).